pkg/logging/http.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+
+	"net/http"
+	"time"
+	"github.com/thanos-io/thanos/pkg/server/http/middleware"
+)
+
+type LogServerMiddleware struct {
+	opts   *options
+	logger log.Logger
+}
+
+// Things for logger
```

Comment:
Do we need this comment?

Reply:
Nope, its more for the TODOs, will remove once the prototype starts working

Reply:
Removed